lively.morphic/tests/components-test.cp.js

```diff
@@ -767,30 +789,21 @@ describe('components', () => {
     expect(inst.get('bob').fill).to.eql(Color.red);
     expect(inst.get('alice').fill).to.eql(Color.blue);
     expect(inst.fill).to.eql(Color.cyan);
+    // this appears to be broken. The overridden master here
+    // needs to also overriden the masters in the sub policies.
+    // what needs to happen is a subPolicy.assignConfig(parent._autoMaster.getSubSpecFor(subSpecName)); // which will update the fields
     inst = part(c2, { name: 'troller', master: T1 });
-    expect(inst.master.overriddenMaster, 'correctly overrides top level master').to.eql(T1.stylePolicy);
+    expect(inst.master._autoMaster, 'correctly overrides top level master').to.eql(T1.stylePolicy);
```

Comment:
:thinking:

Reply:
No the comment stems from a time when this test was in fact broken. It no longer is, so it can go.